lib/rules/prefer-spread.js

```diff
@@ -23,7 +23,8 @@ function isVariadicApplyCalling(node) {
         node.callee.property.name === "apply" &&
         node.callee.computed === false &&
         node.arguments.length === 2 &&
-        node.arguments[1].type !== "ArrayExpression"
+        node.arguments[1].type !== "ArrayExpression" &&
+        node.arguments[1].type !== "SpreadElement"
```

Comment:
If the expression is

```js
foo.apply(null, ...args);
```

Then this would result in a SyntaxError:

```js
foo(......args);
```

The rule only warns when `node.arguments.length === 2`, but when calling `foo.apply(null, ...args)`, the actual number of arguments will be `1 + args.length`, which might not be 2.